src/Files.App/Views/LayoutModes/StandardLayoutMode.cs

```diff
@@ -170,7 +170,7 @@ protected virtual async void FileList_SelectionChanged(object sender, SelectionC
 			SelectedItems = ListViewBase.SelectedItems.Cast<ListedItem>().Where(x => x is not null).ToList();
 
 			if (SelectedItems.Count == 1 && App.AppModel.IsQuickLookAvailable)
-				await QuickLookHelpers.ToggleQuickLook(ParentShellPageInstance, true);
+				await QuickLookHelpers.ToggleQuickLook(SelectedItem.ItemPath, true);
```

Comment:
@cinqmilleans how can I trigger the action here?

Reply:
Declare the service in the class.
`private readonly ICommandManager commands = Ioc.Default.GetRequiredService<ICommandManager>();`

The method simply becomes this. This code checks that the action is IsExecutable, and executes it if possible.
```
protected virtual async void FileList_SelectionChanged(object sender, SelectionChangedEventArgs e)
{
    await commands.LaunchQuickLook.ExecuteAsync();
}
```

Reply:
Can I pass an argument?

Reply:
No, I think you have to use contexts to get the value that you need

Reply:
Changing the selection of an item (without pressing the space key) is supposed to toggle QuickLook but with a different command. I guess we'll need a separate action for this?

Reply:
No. IContentPageContext allows you to be notified when the selection changes, and to know the selection and the parent folder. The action can act accordingly.